.prow.sh

```diff
@@ -1,5 +1,19 @@
 #! /bin/bash -e
+
+# Copyright 2014 The Kubernetes Authors.
```

Comment:
Why is this 2014?

Reply:
Shall I change it to 2019 (sam as in `prow.sh`)?

Reply:
should use 2021